style/icons/all.less

```diff
@@ -56,7 +56,7 @@
 
 // Note: Use tokens instead of strings to be able to do LESS compile-time validation
 .icon-feedback { .x-icon-bg(feedback); }
-.icon-homework { .x-icon-bg(homework); }
+.icon-exercise, .icon-homework, .icon-end { .x-icon-bg(homework); }
```

Comment:
i think the icon representing icon-end is bound to change to be a different icon altogether, so i don't know if that's best at the moment